### ISSUE TITLE
Train and calibrate the integrated BG-NS-JEPA candidate

### ISSUE THREAD

**Issue by Sino-Huang** [Sino-Huang/NovPhy#58]
## Parent

#1 — BG-NS-JEPA model and experimental design

## Why this ticket exists

Issue #41 found that the best available pre-integration evidence is insufficient to freeze #34. Its fixed `h1/continuous` comparator result is local and teacher-forced; it does not measure accumulated rollout error. This ticket supplies the missing integrated candidate, matched stress ablations, and model-level recursive continuous-rollout evidence before the protocol is frozen.

## Frozen design

Before calibration results are inspected, freeze:

- Full 3×3 shared predictor with the ordered-flat micro adapter retained by #13, the reliability estimator retained by #12 used only for symbolic loss gating, and the accepted continuous and macro paths.
- Matched training schedule: seed `20260824`, 1,800 updates, exactly 200 per pair, batch size 32, learning rate `3e-4`, and symbolic weight 1.0.
- Controller seed 10, one aggregation round, and bootstrap seed `20260826`; no outcome-conditioned seed changes.
- Candidate: integrated aggregated joint controller.
- Comparators: the four #9 policies plus the matched-capacity two-head controller.
- Stress ablations: a matched-capacity integrated no-symbol model and an ordered-flat model without the reliability gate.

## Acceptance criteria

- Train and source-bind the integrated candidate and ablations using training-role learned parameters only.
- Use model selection only to freeze configurations and comparators, and calibration only for thresholds, margins, precision, and sensitivity.
- Regenerate exhaustive pair evaluation, compute and violation measurements, trajectory labels, controllers, and six rollout-level calibration records.
- Select the strongest eligible comparator at each declared compute budget; do not compare configurations outside their matched-compute support.
- Report local teacher-forced metrics separately from complete-rollout metrics.
- Test continuous-mode accumulation by recursively applying fixed `h1`, `h5`, and `h15` paths over equal simulated duration. Use only the initial carrier, supplied intervention, and predicted successor carriers; never feed future engine states or labels. Use authoritative endpoints only for scoring. Record requested and effective horizons, error curves, terminal MSE, error AUC, and total compute.
- Do not claim recursive physical-violation labels for predicted carriers. Report that metric unavailable unless a valid prediction-side derivation exists.
- Keep adaptive symbolic recursive planning and multi-shot outcomes out of scope; #56 and #57 own observe-plan-act-replan and gameplay evidence.
- Emit visible foreground logs, a no-write dry-run, immutable machine-readable artifacts, exact validation, and a sufficient or insufficient disposition for #34.
- Keep learned-parser stress tests downstream of #15; they are not prerequisites for this pre-#34 calibration.

## Blocks

- #34 — Freeze the prospective statistical analysis protocol

**Comment by Sino-Huang** [comment]
Implementation is complete and pushed to origin/env at e2e4bf0c51ea6df8840be1cdda96c5e22aa21507.

Implemented:
- full matched-capacity 3x3 candidate using the retained ordered-flat adapter and issue-12 reliability estimator only for micro symbolic-loss gating;
- integrated no-symbol and ordered-flat ungated stress ablations;
- source-bound checkpoints with exactly 1,800 updates and 200 updates per pair;
- regenerated evaluation/measurement/trajectory-label/baseline/controller/one-round aggregation pipeline;
- budget-scoped selection among all four issue-9 policies plus the matched-capacity two-head controller;
- separate local teacher-forced and complete recursive h1/h5/h15 metrics, including requested/effective horizons, error curves, terminal MSE, error AUC, and total compute;
- recursive physical-violation status explicitly unavailable for predicted carriers;
- immutable evidence, compact report, and exact validation modes with foreground progress logs.

Verification completed without opening final evaluation:
- 91 cohort-v2 regression tests passed;
- final no-write dry-run passed across all three variants, all nine pair paths, and 36 recursive rollout records;
- git diff check passed.

Dry-run:
    python -u -m scripts.run_cohort_v2_integrated --dry-run

Long-running production command:
    python -u -m scripts.run_cohort_v2_integrated --implementation-commit e2e4bf0c51ea6df8840be1cdda96c5e22aa21507

Exact validation after production:
    python -u -m scripts.run_cohort_v2_integrated --validate

The production command writes full ignored artifacts under .local-artifacts/issue-58-integrated and the compact report at data/runtime_evidence/issue-58/cohort-v2-integrated-calibration-summary.json. This issue remains open pending that production run, validation, evidence commit, and final disposition.

**Comment by Sino-Huang** [comment]
## Production disposition

The source-bound production run completed and exact validation passed independently under Python 3.13 with the same artifact identity reported by the operator.

### Evidence boundary

- Learned parameters used the training exposure role only.
- Model selection selected configurations and eligible comparators.
- Calibration supplied budgets, thresholds, margins, precision, and sensitivity.
- Six complete calibration rollouts and six complete model-selection rollouts were covered.
- Final evaluation was not accessed.
- All three variants had 3,013,577 trainable parameters and the frozen 1,800-update schedule, exactly 200 updates per pair.

### Local teacher-forced calibration

At policy compute budget 106659.08223346318, the only eligible comparator was matched-capacity two-head. The integrated aggregated joint controller tied it: mean endpoint gain 0.0, bootstrap interval [0.0, 0.0], and physical-violation increase 0.0.

At budget 1398656.0, fixed pair was the strongest eligible comparator. The candidate mean endpoint gain was -0.00044543943863004104 with bootstrap 95 percent interval [-0.000625167984252087, -0.0002922774501996692]. Negative means the candidate was worse locally. Mean authoritative source-endpoint violation increase was 0.00024550443268996455 with interval [0.00011415659000191565, 0.0003768522753780134]. These are local teacher-forced measurements, not recursive predicted-carrier violation labels.

### Complete recursive continuous rollouts

Model selection chose fixed h15 as the strongest recursive continuous horizon by terminal MSE. Calibration means were:

- h1: terminal MSE 19105.552408854168; error AUC 1600434.6127194883
- h5: terminal MSE 398.9628194173177; error AUC 24143.300477127737
- h15: terminal MSE 2.6611214677492776; error AUC 118.76116295269458

Repeated h1 therefore accumulates severe model error. Recursive physical-violation labels remain unavailable because predicted carriers are not authoritative engine frame records.

### Stress ablations

Relative to the gated ordered-flat candidate, mean local calibration degradation was +0.00006572906735970151 for integrated no-symbol and +0.00003532994428595882 for ordered-flat without reliability gating. Positive gaps favor the retained candidate components on this local metric.

### Disposition

Sufficient evidence to freeze issue #34. This means the required calibration evidence is complete; it does not turn the local result into support for the central confirmatory hypothesis. Planning and gameplay remain owned by #56 and #57.

Revisions:
- integrated implementation: e2e4bf0c51ea6df8840be1cdda96c5e22aa21507
- cross-Python exact-validation fix: 556f0d9
- compact-report publication fix: 3a6a392
- compact evidence: 775a820ad086745d11cb1971dd8a84d1477e32e7

Committed report: data/runtime_evidence/issue-58/cohort-v2-integrated-calibration-summary.json

Rerun commands:
    python -u -m scripts.run_cohort_v2_integrated --dry-run
    python -u -m scripts.run_cohort_v2_integrated --implementation-commit e2e4bf0c51ea6df8840be1cdda96c5e22aa21507
    python -u -m scripts.run_cohort_v2_integrated --validate
